contrib/kubernetes/datahub/Chart.yaml

```diff
@@ -4,28 +4,7 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.1
+version: 0.2.1
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: latest #0.3.1
-dependencies:
```

Comment:
You are right. I initially removed this because `helm dep update` doesn't work for datahub since all the subcharts are in the charts directory. Reverting. Going to keep the version update since multiple setup jobs were added.

Reply:
Updated version to 1.1.2 since the change is minor